tests/acceptance/let-helper-test.js

```diff
@@ -94,6 +94,22 @@ describe('Acceptance: let helper', function() {
         });
       });
 
+      it('scopes to outmost scope', () => {
+        andThen(() => {
```

Comment:
We shouldn't need to have any `andThen` in our testcases. If it doesn't work without them, then there is an underlying issue.

I see that there are a bunch in there already, so I'll add an issue to take them out. Does this not work without it?